adagrams/game.py

```diff
@@ -1,11 +1,143 @@
+import random
+
+letter_pool = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    """
+    function to create a random hand of 10 letters
+    no input
+    output is a list of 10 random letters
+    """ 
+    # assign variable names to the letters and max usage num 
+    letter_list, usage_list = list(letter_pool.keys()), list(letter_pool.values())
+    
+    hand = []
+    while len(hand) < 10:
+        position = random.randint(0, len(letter_list)-1)
+        letter_frequency = usage_list[position] # assigns the random int to a usage number
+        random_letter = letter_list[position] # assigns the random int to a letter
+        
+        if hand.count(random_letter) < letter_frequency:
+            add_letter = hand.append(random_letter)
+        else:
+            continue
+    return hand
 
 def uses_available_letters(word, letter_bank):
-    pass
+    """
+    function to check if a letter is in a list
+    input is word (a string of letters), and letter bank (an array of words)
+    output is return True if letter is in letter bank, or return False if letter is not in letter bank
+    """ 
+    # word = word.upper()
+    # count = 0
+    # for letter in word:
+    #     if word.count(letter) > letter_bank.count(letter):
+    #         return False
+    #     elif letter in letter_bank:
+    #         count += 1
+    # if count == len(word):
+    #     return True
+
+    word = word.upper()
+    list = []
+    letter_bank_copy = letter_bank.copy()
+
+    for each_letter in word:
+        if each_letter in letter_bank_copy:
+            list.append(each_letter)
```

Comment:
Right now I don't see the use case for making the `list` and appending things to it. If you remove it from your code it would work the same. Feel free to reply to this comment and explain your reasoning.